packages/shared/src/components/modals/post/ReportCommentModal.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactElement, useState } from 'react';
+import { useMutation } from 'react-query';
+import { Radio } from '../../fields/Radio';
+import { Button } from '../../buttons/Button';
+import { Modal, ModalProps } from '../common/Modal';
+
+export interface Props extends ModalProps {
+  onReport?(): void;
+}
+
+const reportReasons: { value: string; label: string }[] = [
```

Comment:
I think the capitalisation is wrong, but checking quickly on slack.
Generally we don't allow capitals down the sentence.

https://dailydotdev.slack.com/archives/C0592NL7YCU/p1688107441570529

Reply:
Agreed. Was thinking of that too. Thanks for the link to the channel.